src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,35 +1,46 @@
 package com.kodilla.ecommercee.controller;
 import com.kodilla.ecommercee.dto.UserDto;
+import com.kodilla.ecommercee.mapper.UserMapper;
+import com.kodilla.ecommercee.service.UserService;
 import org.springframework.web.bind.annotation.*;
-import java.util.ArrayList;
 import java.util.List;
 
 @CrossOrigin("*")
 @RestController
 @RequestMapping("/v1/ecommercee/user/")
 public class UserController {
 
+    private final UserService userService;
+
+    private final UserMapper userMapper;
+
+    public UserController(UserService userService, UserMapper userMapper) {
+        this.userService = userService;
+        this.userMapper = userMapper;
+    }
```

Comment:
zastanawiam sie nad tym konstruktorem czy on jest tu potrzebny ? Masz nad klasą UserMapper adnotację @Compoment i nad klasą UserService andotacje @Service, wiec bezproblemu możesz je tutaj przekazać przy pomocy adnotacji @Autowired , nie ma potrzeby tworzenia kontruktora.

Reply:
https://blog.marcnuri.com/field-injection-is-not-recommended/

Reply:
Jak najedziesz w Intellij to Ci to samo podpowie. Mozemy zrobic @Autowired tyle ze myslalem, zeby to zrobic zgodnie ze dzisiejszymi standardami.